subworkflows/OrganelleFinder/nextflow.config

```diff
@@ -0,0 +1,96 @@
+manifest {
+    name = 'OrganelleFinder'
+    author = ' Vendela Andersson, William Eulau, Viktor Säde'
+    homePage = ''
+    description = 'Nextflow genomic organelle finder pipeline'
+    mainScript = 'OrganelleFinder.nf'
+    nextflowVersion = '!>=20.01.0'
+    version = '0.2'
+}
+
+params {
+    // Input genome paths
+    genome_assembly = ''
+    reference_mitochondria = ''
+    reference_chloroplast = ''
+    reads_file = ''
+
+    // Output directory
+    outdir = ''
+    input_type = ''
+
+    mit_blast_evalue = ''
+    mit_bitscore = ''
+    mit_significant_gene_matches = ''
+    mit_suspicious_gene_matches = ''
+    mit_max_contig_length = ''
+    mit_min_span_fraction = ''
+
+    chl_blast_evalue = ''
+    chl_bitscore = ''
+    chl_significant_gene_matches = ''
+    chl_suspicious_gene_matches = ''
+    chl_max_contig_length = ''
+    chl_min_span_fraction = ''
+}
+
+
+profiles {
+    standard {
+        process.executor = 'local'
+    }
+
+    singularity {
+        singularity.enabled = true
+    }
+
+    debug { process.beforeScript = 'env' }
+
+    docker {
+        docker.enabled = true
+        docker.runOptions='-u "$( id -u ):$( id -g )"'
+    }
+    uppmax {
+        executor {
+            name = 'slurm'
+        }
+        process {
+            scratch = '$SNIC_TMP'
+        }
+        singularity.enabled = true
+        singularity.envWhitelist = 'SNIC_TMP'
+    }
+
+process {
```

Comment:
This process configuration block is sitting inside the profiles block, so it won't be seen. You've configured nextflow to look for a profile named process here.